### DISCUSSION TITLE
Asset Thumbnails

### DISCUSSION THREAD

**Discussion by crackAT** [statamic/cms#11121] [General]
Hi all, 

i installed and implemented a cloudinary driver for all images but i have troubles with thumbnails. Is it possible to get thumbnails and previews directly from cloudinary or is it only possible via the glide cache? 

Thanks

**Comment by edalzell** [comment]
Only glide at the moment, but there are some plans to switch to a driver-based system: https://github.com/statamic/cms/pull/10204 that could maybe be used in the CP too
